features/fixtures/maze_runner/Assets/Scripts/MobileScenarioRunner.cs

```diff
@@ -168,6 +174,11 @@ public void DoTestAction(string scenarioName)
                 SetUser();
                 LogError();
                 break;
+            case "Java Background Crash":
+                LeaveBreadcrumbString();
+                LeaveBreadcrumbTuple();
```

Comment:
Given the separation of "throw Exception" and "throw Exception with breadcrumbs": should "Java Background Crash" really be leaving breadcrumbs before crashing?

Reply:
Removed the unnecessary calls to the breadcrumb methods